### DISCUSSION TITLE
My github student developer pack approved but does not working

### DISCUSSION THREAD

**Discussion by YAVUZINAN111** [community/community#55426] [GitHub Education]
### Hi everyone,

hello, my account has been approved and I have received an e-mail, but when I click the yes I am a student button to get the student developer pack, it takes me back to the page (https://education.github.com/benefits?type=student) .

**Comment by MrDaPoyo** [comment]
Me too, I've already been waiting for more than 3 days, the supposed time for the activation.

**Comment by Rebooting-Me** [comment]
Why is this issue closed? Did anybody find the answer to this? I too am facing the same problem.

**Reply by MrDaPoyo** [reply]
Yeah! You just need to wait.

**Reply by moxSedai** [reply]
How long?

**Comment by MrDaPoyo** [comment]
> How long?
A few days
